builds/.env.sample

```diff
@@ -0,0 +1,8 @@
+# ENV_FILE is the name of the file that contains the environment variables
+# for the timelock. This file should either be located in the project root
+# or be a relative path from the project root.
+#
+# i.e. If the file is located in the project root, then the value of ENV_FILE
+# should be "timelock.env". If the file is located in a subdirectory, then the
+# value of ENV_FILE should be "subdirectory/timelock.env".
+ENV_FILE="timelock.env"
```

Comment:
used in the docker compose file to specify the source config file that will be mounted in the container at `/app/timelock.env`